src/containers/PersonContainer/WriteBiography.tsx

```diff
@@ -0,0 +1,86 @@
+import Person from "../../models/Person";
+import Role from "../../models/Role";
+import MatterSponsor from "../../models/MatterSponsor";
+import ordinalSuffix from "../../utils/ordinalSuffix";
+import { filterRolesByTitle, ROLE_TITLE, getMostRecentRole } from "../../models/util/RoleUtilities";
+
+function WriteBiography(
+  person: Person,
```

Comment:
Since this is not a component, change the name to `writeBiography`?

Reply:
done